modules/monitoring-example-remote-write-authentication-settings.adoc

```diff
@@ -124,10 +123,9 @@ data:
 <1> The name of the `Secret` object that contains the authentication credentials.
 <2> The key that contains the username  in the specified `Secret` object.
 <3> The key that contains the password in the specified `Secret` object.
-====
 
-.Sample YAML for authentication with a bearer token using a `Secret` Object
-====
+== Sample YAML for authentication with a bearer token using a `Secret` Object
```

Comment:
[All headings in module files](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings) take an anchor ID.